src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -1034,6 +1174,50 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 *  Magento no longer adds a `form_key` field to POST forms that had external action URLs. (*External action URLS* are URLS that do not belonging to shop's base URL.) *Fix submitted by Mateusz Krzeszowiak in pull request [25336](https://github.com/magento/magento2/pull/25336)*. [GitHub-23382](https://github.com/magento/magento2/issues/23382)
 
+<!--- MC-18190-->
+
+*  The dictionary was removed from the `zxcvbn.js` library, and the following performance improvements have resulted: 1) The size of the `zxcvbn` library has been reduced from 395 KB to 11.3 KB on customer registration, customer edit, and customer forgot password pages. 2) The time required for asynchronously loading this library has been reduced by 90%.
+
+<!--- ENGCOM-5634-->
+
+*  The `scopeData()` method now returns a `DateTime` value that is scoped to the specified store locale. Previously, this method was not fully implemented.  *Fix submitted by Eden Duong in pull request [23693](https://github.com/magento/magento2/pull/23693)*. [GitHub-23359](https://github.com/magento/magento2/issues/23359)
+
+<!--- ENGCOM-5683-->
+
+*  The `getAttributeRawValue` method now returns a store-specific value even when there is no default value. Previously, no store value was returned when a default value was not present. *Fix submitted by semajeg in pull request [23369](https://github.com/magento/magento2/pull/23369)*. [GitHub-16382](https://github.com/magento/magento2/issues/16382)
+
+<!--- ENGCOM-5554-->
+
+*  The performance of the `ProductMetadata::getVersion` method has been improved as a result of adding the caching of he product version. This method is called by many third-party extensions to determine the version of Magento. *Fix submitted by David Verholen in pull request [24030](https://github.com/magento/magento2/pull/24030)*. [GitHub-24025](https://github.com/magento/magento2/issues/24025)
+
+<!--- ENGCOM-6119-->
+
+*  You can now add products with custom options of all types to the shopping cart. *Fix submitted by Alexandr Skrashuk in pull request [25055](https://github.com/magento/magento2/pull/25055)*. [GitHub-24726](https://github.com/magento/magento2/issues/24726)
+
+<!--- ENGCOM-6186-->
+
+*  Decimal numbers have been added to the Sample File in Import CSV section. Previously, using the **Download Sample File** option on the product’s import CSV page created a sample CSV file with a weight of 1, which was of limited use. *Fix submitted by Cristian Sanclemente in pull request [25317](https://github.com/magento/magento2/pull/25317)*. [GitHub-23920](https://github.com/magento/magento2/issues/23920)
+
+<!--- ENGCOM-6171-->
+
+*  A deprecated method in `\Magento\MysqlMq\Model\Driver\Exchange` has been replaced. Previously, this method caused an exception. *Fix submitted by Tjitse in pull request [25289](https://github.com/magento/magento2/pull/25289)*. [GitHub-21904](https://github.com/magento/magento2/issues/21904)
```

Comment:
You have to say which method was replaced!

Reply:
will ask community member

Reply:
added the method name - thanks